src/systemctl/systemctl-list-units.c

```diff
@@ -403,19 +402,18 @@ static int output_sockets_list(struct socket_info *socket_infos, unsigned cs) {
                                 return table_log_add_error(r);
 
                 }
-
```

Comment:
If you touch here, then could you also update `if (cs)` -> `if (cs > 0)` in the above?

Reply:
I noticed the condition `if (cs)` can be dropped now.

Reply:
Done.